src/leaflike/bookmarks/core.clj

```diff
@@ -0,0 +1,46 @@
+(ns leaflike.bookmarks.core
+  (:require [leaflike.bookmarks.validator :refer [valid-bookmark?
+                                                  id?]]
+            [leaflike.bookmarks.db :as bm-db]
+            [leaflike.user.db :as user-db]
+            [leaflike.user.auth :refer user-session]
+            [leaflike.utils :as utils]))
+
+(defn- get-user
+  []
+  (when-not (nil? @user-session)
```

Comment:
Can this be simplified with a when-let instead https://clojuredocs.org/clojure.core/when-let?